animation_nodes/nodes/material/object_material_output.py

```diff
@@ -1,10 +1,15 @@
 import bpy
+from bpy.props import *
 from ... base_types import AnimationNode, VectorizedSocket
 
 class ObjectMaterialOutputNode(bpy.types.Node, AnimationNode):
     bl_idname = "an_ObjectMaterialOutputNode"
     bl_label = "Object Material Output"
 
+    appendMaterials: BoolProperty(name = "Append Materials", default = False,
+        description = "This option allow to add custom materials",
```

Comment:
`Append input material(s) to the object's materials instead of overwriting them`

Reply:
Sorry, I didn't understand this point?

Reply:
I mean change the description to this.

Reply:
Okay.